ztp/gitops-subscriptions/argocd/deployment/resource-hook-cluster-role.yaml

```diff
@@ -0,0 +1,147 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: converter
```

Comment:
Minor request. Maybe update the name to be more reflective of the function, eg site-converter

Reply:
done